README.md

```diff
@@ -174,45 +174,45 @@ Support for OpenGL accelerated rendering is made possible through the work our D
 
 All of these in-flight contributions are kept in Microsoft mirror repos. We keep these mirrors up to date with upstream releases and stage our WSLg changes in those repos. WSLg pulls and builds code from these mirror repos as part of our Insider WSLg Preview releases. These mirrors are public and accessible to everyone. Curious developer can take a peek at early stages of our contribution by looking at code in those mirrors, keeping in mind that the final version of the code will likely look different once the contribution reaches the upstream project and is adapted based on the feedback receives by the various project owners. All of our mirrors follow the same model. There is a **main** branch which correspond to the upstream branch at our last synchronization point. We update the **main** branch from time to time to pick update from the upstream project. There is also a **working** branch that contains all of our in-flight changes. WSLg is built using the **working** branch from each of the mirror projects.
 
-Which projects WSLg maintains mirror for will change overtime as in-flight contributions evolve. Once some contributions are upstream, it may not longer be necessary to maintain a mirror at which point it will be removed and WSLg will start to leverage the upstream version of the component directly. As we light up new functionality in WSLg, new mirrors may get introduce to stage contributions to new components. As such as expect the list of mirrors to change overtime.
+Which projects WSLg maintains mirrors for will change over time as in-flight contributions evolve. Once some contributions are upstream, it may no longer be necessary to maintain a mirror, at which point it will be removed and WSLg will start to leverage the upstream version of the component directly. As we light up new functionality in WSLg, new mirrors may be introduced to stage contributions to new components. As such, expect the list of mirrors to change overtime.
 
-At this point in time we have the following project mirrors for currently in-flight contributions.
+At this point in time, we have the following project mirrors for currently in-flight contributions.
 
 | Project | Upstream Repo | WSLg Mirror |
 |---|---|---|
 | Weston | https://github.com/wayland-project/weston | https://github.com/microsoft/Weston-mirror|
 | FreeRDP | https://github.com/FreeRDP/FreeRDP | https://github.com/microsoft/FreeRDP-mirror |
 | PulseAudio | https://github.com/pulseaudio/pulseaudio | https://github.com/microsoft/PulseAudio-mirror |
 
-Here's a high level overview of the currently in-flight contributions to each project contained within these mirrors.
+Here is a high level overview of the currently in-flight contributions to each project contained within these mirrors:

```

Comment:
I'd change the sentence introduction so that the period can stay, because having a colon like this before a list with headers before each paragraph seems suboptimal.
```suggestion
The following is a high level overview of the currently in-flight contributions to each project contained within these mirrors.
```